### ISSUE TITLE
How can I shorten too long command in yml file?

### ISSUE THREAD

**Issue by nuhkoca** [adrienverge/yamllint#211]
Hello,

Thanks for such a great tool, it helps me a lot. I am a newbie to YML and YAML. I have below command in my file

```
command: |
      mkdir -p ~/junit/
      find . -type f -regex ".*/build/test-results/.*xml" -exec cp {} ~/junit/ \;
```

Yamllint throws:

> error    line too long (87 > 80 characters)  (line-length)

Okay, I can shorten this in a valid format but and now my CI tool throws an error:

```
mkdir: invalid option -- 't'
Try 'mkdir --help' for more information.
```

What is the most efficient way to shorten such command in a yml file? Thanks in advance.

**Comment by adrienverge** [comment]
Hi @nuhkoca,

This is not really related to yamllint, but here are a few examples of how you can do to split a long line:
```yaml
command: >
  mkdir -p ~/junit/

  find . -type f -regex ".*/build/test-results/.*xml"
  -exec cp {} ~/junit/ \;
```
```yaml
command:
  'mkdir -p ~/junit/

   find . -type f -regex ".*/build/test-results/.*xml"
          -exec cp {} ~/junit/ \;'
```
```yaml
command: "mkdir -p ~/junit/\n\
          find . -type f -regex \".*/build/test-results/.*xml\"
                 -exec cp {} ~/junit/ \\;"
```

There are other options, I recommend reading https://stackoverflow.com/a/21699210 or playing with https://yaml-multiline.info/.

I'll close this issue because this it out of the scope of linting, but feel free to comment further.

PS: You can also keep the long line, and just change yamllint configuration from 80 to 100 chars, or disable the `line-length` rule. See https://yamllint.readthedocs.io/en/stable/configuration.html#extending-the-default-configuration.

**Comment by nuhkoca** [comment]
Thanks for your help @adrienverge  it works now.

**Comment by RachaelSMathew** [comment]
Hi,

I have a yaml file in Azure Devops that is really long and contains some sub-pipelines. In order to make this yaml file more easier to understand, I am trying to place these sub-pipelines in separate files and run it from the main pipeline, is this possible?
